acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNodeConfigBuilder.java

```diff
@@ -317,6 +317,18 @@ public TekuNodeConfigBuilder withWritableKeystorePath(
     return this;
   }
 
+  public TekuNodeConfigBuilder withDefaultLogging() {
+    configMap.remove("log-destination");
+    return this;
+  }
+
+  public TekuNodeConfigBuilder withBadKeystorePath() {
```

Comment:
I don't think it is worth polluting our DSL with such a specific usage, specially being a "negative" usage that is unlikely to see any reuse.

I'd argue for adding (more flexible way of setting up the path: 
```
withValidatorKeysPath(String path) {
  configMap.put("validator-keys", path);
}
```

And then, when creating your node, you can use:
```
final TekuBeaconNode beaconNode =
        createTekuBeaconNode(
            TekuNodeConfigBuilder.createBeaconNode()
                .withDefaultLogging()
                .withValidatorKeysPath("/abcde:/abcdef")
                .withInteropModeDisabled()
                .withInitialState(genesis)
                .build());
```